recipes/pflare/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pflare" %}
+{% set version = "1.25.0" %}
+{% set petsc_version = "3.24.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: "https://github.com/PFLAREProject/PFLARE/archive/refs/tags/v{{ version }}.tar.gz"
+    sha256: befb361b39c7601a8ca6f148369313f5755b238d5f6a4cbf91b19b23c93e8952
+    folder: pflare_src
+  - url: "https://gitlab.com/petsc/petsc/-/archive/v{{ petsc_version }}/petsc-v{{ petsc_version }}.tar.gz"
+    sha256: de0bb8c515e838acdfa37da4fe7cf883f89a9e58ef32702469bc6906ef359fab
+    folder: petsc_src
```

Comment:
Can you explain why are you downloading the petsc source code if petsc is installed by conda-forge as a dependency?

Reply:
I see. Perhaps @conda-forge/petsc would be interested in provide their feedback here?

Reply:
Specifically its the Mvctx (the SF) in Mat_MPIAIJ (defined in src/mat/impls/aij/mpi/mpiaij.h) that PFLARE needs access to. 

I've just opened up an issue in the PETSc gitlab (https://gitlab.com/petsc/petsc/-/issues/1827) to see if there is any way access to that might be added explicitly (or if I missed something). If not perhaps the Mat_MPIAIJ could be added to the private headers (having a look through it looks like Mat_MPIDense is already in include/petsc/private/matimpl.h).